netmiko/centec/centec_os.py

```diff
@@ -1,31 +1,19 @@
 """Centec OS Support"""
 from netmiko.cisco_base_connection import CiscoBaseConnection
 import time
-import re
 
 
 class CentecOSBase(CiscoBaseConnection):
     def session_preparation(self):
         """Prepare the session after the connection has been established."""
         self._test_channel_read(pattern=r"[>#]")
         self.set_base_prompt()
-        self.disable_paging()
-        self.set_terminal_width(command="terminal length 0")
+        self.enable()
```

Comment:
Was this `enable()` call needed for some reason?

Reply:
hi ktbyers
Thank you for checking the code for me
centec_os.py is no longer a problem. Actually, you don't need to add enable (), but actually I want to add centec to the list of platforms. Could you please help to add this?

Reply:
Sorry,
has been added to the platform,
But https://ktbyers.github.io/netmiko/docs/netmiko/index.html has not been added here